implementations/rust/ockam/ockam_api/src/logs/logging_configuration.rs

```diff
@@ -83,22 +83,27 @@ impl LoggingConfiguration {
 
     pub fn set_crates(self, crates: &[&str]) -> LoggingConfiguration {
         LoggingConfiguration {
-            crates: crates.iter().map(|c| c.to_string()).collect(),
+            crates: Some(crates.iter().map(|c| c.to_string()).collect()),
             ..self
         }
     }
 
     pub fn env_filter(&self) -> EnvFilter {
-        let builder = EnvFilter::builder();
-        builder
-            .with_default_directive(self.level().into())
-            .parse_lossy(
-                self.crates()
-                    .iter()
-                    .map(|c| format!("{c}={}", self.level()))
-                    .collect::<Vec<_>>()
-                    .join(","),
-            )
+        match &self.crates {
+            Some(crates) => {
+                let builder = EnvFilter::builder();
+                builder
+                    .with_default_directive(self.level().into())
+                    .parse_lossy(
+                        crates
+                            .iter()
+                            .map(|c| format!("{c}={}", self.level()))
+                            .collect::<Vec<_>>()
+                            .join(","),
+                    )
+            }
+            None => EnvFilter::default().add_directive(LevelFilter::INFO.into()),
```

Comment:
Shouldn't this use the `self.level()` function instead of `LevelFilter::INFO`?

Reply:
Yes, indeed. I fixed it, thanks.